src/app/_pages/join/join.page.scss

```diff
@@ -0,0 +1,60 @@
+mat-grid-tile {
+    background: #0277bd;
+    border-radius: 5px;
+}
+
+button { 
+    position: absolute; 
+    bottom: 5%;
+    right: 5%;
+    border: 0px solid #d6d6d6;
+    width: 150px;
+    height: 50px;
+}
+
+mat-selection-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    column-gap: 1em;
+    padding-right: 1.5em;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+}
+
+mat-selection-list>* {
+    flex: 0 0 20%;
+}
+
+.mat-list-option {
+    background: #383838;
+    margin-top: 10px; 
+    margin-bottom: 10px; 
+    margin-right: 10px; 
+    margin-left: 10px; 
+    border-radius: 5px;
+    align-content: flex-start;
+    text-align: center;
+    height: 15vh;
+}
+
+.mat-list-option.mat-list-item-focus, .active, mat-list-option[aria-selected="true"] {
+    background: #0277bd !important;
+}
+
+.mat-list-option:hover {
+    background-color: rgba($color: #0277bd, $alpha: .5);
+}
+
+:host {
+    ::ng-deep.mat-pseudo-checkbox{
+      display: none !important;
```

Comment:
if you add a class to mat-selection-list (like 'hide-checkbox') you can probably avoid using :host, ::ng-deep, and !important, but you may have to move it to src/global.scss, so it applies to the element properly, but will also make it re-usable.